Sources/Sentry/SentryNetworkTracker.m

```diff
@@ -16,6 +25,7 @@
 
 @property (nonatomic, assign) BOOL isNetworkTrackingEnabled;
 @property (nonatomic, assign) BOOL isNetworkBreadcrumbEnabled;
+@property (nonatomic, assign) BOOL isCaptureFailedRequests;
```

Comment:
For me, its a bit confusing that this SDK has some properties from the options, but some properties are accessed directly from the options, such as `tracePropagationTargets`.
Why do we have a mix of approaches?

Reply:
IMO we should only use options, then we would only need a SentryOption property, but I dont know if there is a reason behind this.

Reply:
Something for @philipphofmann

Reply:
I agree to avoid 'SentrySDK.options' but rather have an options parameter in the constructor. Instead of having multiples parameters.